package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "cake-task",
+  "version": "1.0.0",
```

Comment:
Do we need to update this version before publishing?

Reply:
@patriksvensson modified the build.cake script to version package.json to be consistent with the other versioning locations